fastlane/lib/fastlane/runner.rb

```diff
@@ -20,15 +20,15 @@ def full_lane_name
     # @param parameters [Hash] The parameters passed from the command line to the lane
     # rubocop:disable Metrics/AbcSize
     def execute(lane, platform = nil, parameters = nil)
-      raise "No lane given" unless lane
+      UI.user_error!("No lane given") unless lane
```

Comment:
I wonder if this should be `crash!` instead? seems like you might want a trace of how you got here?